e2e/pnpm_workspace/BUILD.bazel

```diff
@@ -21,9 +22,22 @@ sh_test(
 )
 
 build_test(
-    name = "aliases",
```

Comment:
See the diff in pnpm_workspace_rerooted where `aliases` was moved. 

All the `vendored` stuff in `pnpm_workspace` is stuff that was out of sync with `pnpm_workspace_rerooted` and just copied over (maybe removed `root[:/]` in a few places).